src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -192,7 +192,15 @@ export default connect(state => {
       return (
         <FieldTypeText
           name={name}
-          label={label}
+          label={
+            <>
+              {" "}
```

Comment:
Lets use `;nbsp;` I feel they are more consistent and clearly communicate the intent. Sometimes pretty inserts spacing and this could then be viewed as a mistake and someone would attempt to correct in the future.